pkg/util/cert/cert.go

```diff
@@ -138,7 +138,7 @@ func GenerateSelfSignedCertKey(host string, alternateIPs []net.IP, alternateDNS
 	template := x509.Certificate{
 		SerialNumber: big.NewInt(1),
 		Subject: pkix.Name{
-			CommonName: fmt.Sprintf("%s@%d", host, time.Now().Unix()),
+			CommonName: host,
```

Comment:
Didn't know CN can be nearly anything (https://tools.ietf.org/html/rfc6125). Will restore the old behaviour.